.github/workflows/miri.yml

```diff
@@ -14,15 +14,15 @@ jobs:
     name: Miri
     runs-on: ubuntu-latest
     env:
-      RUSTUP_TOOLCHAIN: nightly-2024-05-24
```

Comment:
This is more of a question here but to update this here, what do we need to base upon?

Reply:
There is not a hard rule, we just need a version that's newer than the msrv and it needs to be fixed so a potential bug on a new nightly doesn't cause a spurious failure.